plugin/core/types.py

```diff
@@ -89,39 +89,124 @@ def cancel_pending(self) -> None:
             self._current_id = -1
 
 
+def read_bool_setting(settings_obj: sublime.Settings, key: str, default: bool) -> bool:
+    val = settings_obj.get(key)
+    if isinstance(val, bool):
+        return val
+    else:
+        return default
```

Comment:
Just a style but maybe more compact:
```suggestion
    return val if isinstance(val, bool) else default
```

Reply:
This function is removed but there are still two other that could use this style.